marketplace/sensitive-language/src/index.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: "Avenir Next W01";
+  src: url("https://static.contentful.com/app/2cd55546-ec00-4af9-aeca-4a3cd186da53.woff2")
```

Comment:
don't use avenir in open-source

Reply:
Will remove it 👍 

@m10l I guess this answers our discussion